scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -126,7 +126,7 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
             if (customSort) {
                 scope.customSortOptions = {
                     ...customSort,
-                    [DEFAULT_SORT_FIELD]: {label: 'Default'},
+                    [DEFAULT_SORT_FIELD]: {label: gettext('Default')},
```

Comment:
This won't work. You need it to be `[DEFAULT_SORT_FIELD]: {label: (gettext) => gettext('Default')},`

Reply:
Sorry, I thought this was a config... It's not, so it's fine.

Reply:
Yeah right here the `customSort` object has all the labels translated already